### PR TITLE
SF-1142: Add CDN release jobs to workflow

### PR DESCRIPTION
**Pull request by jmbtutor**
The supporting JavaScript files were copied almost verbatim from storefront-presets.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }),
+  ].concat(ENV === PROD ? [new UglifyJSPlugin({ sourceMap: true })] : []),
+
+  module: {
+    rules: [{
+      test: path.resolve(__dirname, 'presets', 'core.js'),
+      use: { loader: 'expose-loader', options: 'storefront' }
+    }, {
+      resource: [
+        // /node_modules\/@storefront\/.*\.js$/,
```
Comment (jrmykolyn) [inline webpack.config.js]: Is this required?

File: presets/package-versions.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+module.exports = [
```
[the remark this answers is not in the document]
Comment (jmbtutor) [reply presets/package-versions.js]: This file is imported as part of `core.js`, which is included in the resulting bundle. Will reading directly from the filesystem be resolved before it makes it into the bundle?
https://github.com/groupby/storefront/pull/19/files#diff-2f77dc81edaf077954bc0165fa8b4221R5

File: webpack.config.js (hunk 1 above)
Comment (jmbtutor) [reply webpack.config.js]: This was in the file in storefront-presets, but it appears to be left over from an early iteration. I think we can safely remove it.

File: presets/package-versions.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jrmykolyn) [reply presets/package-versions.js]: Noted, lets proceed with the current solution.

File: webpack.config.js (hunk 1 above)
Comment (jrmykolyn) [reply webpack.config.js]: Sounds good.

**Pull request merged**